src/main/java/edu/hm/hafner/analysis/registry/CrossCoreEmbeddedStudioDescriptor.java

```diff
@@ -0,0 +1,26 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.CrossCoreEmbeddedStudioParser;
+
+/**
+ * A descriptor for CrossCore Embedded Studio from Analog Devices.
+ */
+public class CrossCoreEmbeddedStudioDescriptor extends ParserDescriptor {
+    private static final String ID = "crosscore-embedded-studio";
```

Comment:
This ID `"crosscore-embedded-studio"` will be used in the UI later on in the warnings plugin. Is it ok for you or do you want to shorten it to something like `crosscore`?